engine/dlib/src/dlib/buffer.cpp

```diff
@@ -616,4 +621,70 @@ namespace dmBuffer
         buffer->m_ContentVersion++;
         return RESULT_OK;
     }
+
+
+    Result UpdateBounds(HBuffer hbuffer, dmhash_t stream_name) {
+        Buffer* buffer = GetBuffer(g_BufferContext, hbuffer);
+        if (!buffer) {
+            return RESULT_BUFFER_INVALID;
+        }
+        if (buffer->m_ContentVersion == buffer->m_BoundingContentVersion) {
+            return RESULT_OK; // nothing to do - already calculated bounding box in m_BoundingCube
+        }
+        Buffer::Stream* stream = GetStream(buffer, stream_name);
+        if (stream == 0x0) {
+            return RESULT_STREAM_MISSING;
```

Comment:
How is the missing stream case handled?
Will the bounding info be set so that it's still visible?

Reply:
This has been moved to comp_mesh.cpp and a an error has been reported. 

Closing this as it's out of date.